src/data/downloadMetadata.ts

```diff
@@ -2708,10 +2708,10 @@ export const dataPages: DownloadMetadata = {
     featureServiceHost: 'https://roads.udot.utah.gov/server/rest/services/Public/',
     mapServiceId: 'Freeway_Exits',
   },
-  'UDOT Routes ALRS': {
+  'Utah UDOT Routes ALRS': {
```

Comment:
If you make changes here, you should probably make the same change in the internal meta table.

Reply:
true. i'm gonna revert this change, to not kick off a separate set of issues.